compose.yaml

```diff
@@ -68,3 +54,27 @@ services:
       - "1080:1080"
       - "1025:1025"
     restart: unless-stopped
+  cypress:
+    ipc: host
```

Comment:
Interesting.  Hadn't come across this docker setting before.  Looks useful to know about for some situations. :smile:

https://docs.docker.com/reference/cli/docker/container/run/#ipc